docs/guides/validation/validation_actions/how-to-update-data-docs-as-a-validation-action.md

```diff
@@ -2,4 +2,80 @@
 title: How to update Data Docs as a Validation Action
 ---
 
-This article is a stub.
+import Prerequisites from '../../../guides/connecting_to_your_data/components/prerequisites.jsx';
+
+This guide will explain how to use a Validation Action to update Data Docs sites with new validation results from Validation Operator runs.
+
+:::note Prerequisites
+ This how-to guide assumes you have already:
+
+ - [How to add a Validation Operator](../how-to-add-a-validation-operator)
+ - Set up a `.py` class: `great_expectations.validation_operators.validation_operators.ActionListValidationOperator` **or**
+ - Set up a `.py` class: `great_expectations.validation_operators.validation_operators.WarningAndFailureExpectationSuitesValidationOperator`
+ - Created at least one Expectation Suite.
+ - Created at least one [Checkpoint](../checkpoints/how-to-create-a-new-checkpoint). You will need it in order to test that your new Validation Operator is working.
+
+Steps
+------
+
+1. **Update the action_list key in your Validation Operator config.**
+
+  Add the ``UpdateDataDocsAction`` action to the ``action_list`` key of the ``ActionListValidationOperator`` or ``WarningAndFailureExpectationSuitesValidationOperator`` config in your ``great_expectations.yml``. This action will update all configured Data Docs sites with the new validation from the Validation Operator run.
+
+  :::note Note:
+
+    The ``StoreValidationResultAction`` action must appear before this action, since Data Docs are rendered from validation results from the store.
+
+  ```yaml
+
+    validation_operators:
+      action_list_operator: # this is a user-selected name
+        class_name: ActionListValidationOperator
+        action_list:
+        - name: store_validation_result # this is a user-selected name
+          action:
+            class_name: StoreValidationResultAction
+        - name: update_data_docs # this is a user-selected name
+          action:
+            class_name: UpdateDataDocsAction
+  ```
```

Comment:
```suggestion
  ```yaml
validation_operators:
  action_list_operator: # this is a user-selected name
    class_name: ActionListValidationOperator
    action_list:
      - name: store_validation_result # this is a user-selected name
         action:
           class_name: StoreValidationResultAction
      - name: update_data_docs # this is a user-selected name
         action:
           class_name: UpdateDataDocsAction
  ```
```